src/components/related/related.css

```diff
@@ -0,0 +1,248 @@
+  font-weight: 700;
+}
+.small-title{
+  font:  1.9em "Fira Sans", sans-serif;
+  font-weight: 700;
+}
+
+.button{
+  text-align: right;
+}
+
+.product-info{
+  margin-top: 13px;
+  margin-bottom: 5px;
+}
+
+.category{
+  font:  1.2em "Fira Sans", sans-serif;
+  font-weight: 700;
+}
+
+
+/*
+========================
+        Carousel
+======================== */
+.carousel {
+  background-color: rgb(0, 0, 0, 0);
+  /* display: flex;
+  flex-direction: row;
+  flex-wrap: wrap;
+  justify-content: space-around;
+  align-items: baseline; */
+
+  /* flex-wrap: nowrap; */
+  /* width: 80%; */
+  overflow: auto;
+  scroll-behavior: smooth;
+  width: 85%;
+  overflow-x: hidden;
+  white-space: nowrap;
+}
+
+
+/*
+========================
+     Related Card
+======================== */
+.related-list{
+  background-color: rgb(0, 0, 0, 0);
+  display: flex;
+  flex-direction: row;
+  flex-wrap: nowrap;
+  justify-content: space-around;
+  align-items: baseline;
+  width: 100%;
+}
+
+.related-card{
+  border: solid 3px black;
+  /* position: relative;
+  flex-wrap: nowrap; */
+  display: inline-block;
+  padding: 25px 15px;
+  background-color: white;
+  height: 550px;
+  width: 380px;
+  margin: 15px 15px;
+  box-shadow: 1px 1px 2px black;
+  text-align: justify;
+}
+
+
+/*
+========================
+        Outfit
+======================== */
+ .outfit-list{
+   background-color: rgb(0, 0, 0, 0);
+   display: flex;
+   flex-direction: row;
+   flex-wrap: wrap;
+   justify-content: flex-start;
+  }
+
+  .outfit-card{
+    border: solid 3px black;
+    /* position: relative;
+    flex-wrap: nowrap; */
+    display: inline-block;
+    padding: 25px 15px;
+    background-color: white;
+    height: 550px;
+    width: 380px;
+    margin: 15px 15px;
+    box-shadow: 1px 1px 2px black;
+    text-align: justify;
+  }
+
+  /*
+========================
+   Comparison Modal
+======================== */
+.modal {
+  width: 500px;
+  background: white;
+  border: 1px solid #ccc;
+  box-shadow: -2rem 2rem 2rem rgba(0, 0, 0, 0.2);
+  filter: blur(0);
+  visibility: visible;
+  position: absolute;
+  z-index: 1;
+}
+
+.modal h3 {
+  border-bottom: 1px solid #ccc;
+  padding: 1rem;
+  margin: 0;
+}
+
+.modal .content {
+  padding: 1rem;
+}
+
+.modal .actions {
+  border-top: 1px solid #ccc;
+  background: #eee;
+  padding: 0.5rem 1rem;
+}
+
+.modal .actions button {
+  border: 0;
+  background: #78f89f;
+  border-radius: 5px;
+  padding: 0.5rem 1rem;
+  font-size: 0.8rem;
+  line-height: 1;
+}
+.modal .actions button:hover{
+  cursor: pointer;
+}
+
+
+.comparison{
+  display: flex;
+  /* width:100%;
+  height:100%; */
+  z-index: 3;
+}
+  .left-comparison{
+    float: left;
+    width: 30%;
+    margin: 0 2.5% 0 0;
+    height: 100%;
+  }
+  .middle-comparison{
+
+    float: left;
+    width:30%;
+    margin-left:2.5%;
+    margin-right:5%;
+    height:100%;
+  }
+
+  .right-comparison{
+
+    overflow:hidden;
+    height:100%;
+  }
```

Comment:
I like the organization you have on your css. I think there is potential for condensing some of this but I have never really figured out a good way to do condense css. (Condensing it in relation to repeating code)

Reply:
I think the best way to organize the css file might be, divide your widget into server components and create folder to each of them. Each component has its own css file.